ruby-challenges.rb

```diff
@@ -31,3 +49,16 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome(string)
+    new_string = string.downcase
+    if new_string.reverse == new_string
+        "#{new_string} is a palindrome"
+    else 
+        "#{new_word} is not a palindrome"
```

Comment:
On Line 58, Think you meant new_string vice new_word. However, to give you the expected output as shown on the question, you need to reconsider what word you are performing the string interpolation.